agents/gce/fence_gce.py

```diff
@@ -41,6 +41,21 @@
 
 METADATA_SERVER = 'http://metadata.google.internal/computeMetadata/v1/'
 METADATA_HEADERS = {'Metadata-Flavor': 'Google'}
+INSTANCE_LINK = 'https://www.googleapis.com/compute/v1/projects/{}/zones/{}/instances/{}'
+
+def run_on_fail(options):
+	if "--runonfail" in options:
+		command = options["--runonfail"]
+		logging.info("Due to failure, running command: {}".format(command))
+		os.system(command)
```

Comment:
You should probably use `run_command()` from the fencing library instead of `os.system()` here and the other case where it's used below:
https://github.com/ClusterLabs/fence-agents/blob/5ce6098244fd2116ca1cc3a219235681c22457c8/lib/fencing.py.py#L1085

Reply:
Thanks! I updated the `os.system()` calls to `run_command()`